concent_api/core/message_handlers.py

```diff
@@ -1134,6 +1158,12 @@ def update_subtask(
 
     next_deadline_datetime = parse_timestamp_to_utc_datetime(next_deadline) if next_deadline is not None else None
 
+    if task_to_compute is not None and subtask.task_to_compute is not None:
+        validate_all_messages_identical([
```

Comment:
Isn't that already done in every view handler ?

Reply:
it was. Done